plaidml/core/ffi.h

```diff
@@ -82,6 +98,8 @@ void plaidml_string_free(  //
 typedef struct {
   size_t code;
   plaidml_string* msg;
+  const char* origin_file;
+  uint32_t origin_line;
```

Comment:
Seems like maybe we don't need this since the `call` wrappers already have the `loc` available directly.

Reply:
This would be necessary if we want to somehow include file/line info in the originating throws that occur in the c++ AST code. But since you're just wrapping the exception from the caller side, then we probably don't need to pass this info down.